src/components/Calculator.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+
+const Calculator = () => {
```

Comment:
- [ ] Kindly follow the design guidelines for the calculator component.

| Your design | Guidelines |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/204842731-7497c575-613e-4f55-9aad-81944db74a52.png) | ![image](https://user-images.githubusercontent.com/24703171/204842279-e0c170d8-aa4a-4127-bfcd-8c5d76e9a4e3.png) |